src/parser/Statement.ts

```diff
@@ -2130,6 +2125,38 @@ export class TryCatchStatement extends Statement {
     public walk(visitor: WalkVisitor, options: WalkOptions) {
         if (this.tryBranch && options.walkMode & InternalWalkMode.walkStatements) {
             walk(this, 'tryBranch', visitor, options);
+            walk(this, 'catchStatement', visitor, options);
+        }
+    }
+}
+
+export class CatchStatement extends Statement {
+    constructor(
+        public catchToken: Token,
```

Comment:
I've been trying to store all tokens in a single `tokens` property. can you do the same? See [enum member statement](https://github.com/rokucommunity/brighterscript/blob/b222bce315bffa49d28a3576e3991e86d9df8213/src/parser/Statement.ts#L2181) for an example. The `catchToken` and `exceptionVariable` should both go into `.tokens`